frontend/www/js_templates/provision_cloud.js

```diff
@@ -232,10 +279,10 @@ function loadSelectedEndpointList() {
         <tbody>
           ${peerings}
           <tr id="new-peering-form-${index}">
-            <td><input class="form-control bgp-asn" type="text" /></td>
-            <td><input class="form-control your-peer-ip" type="text" /></td>
+            <td><input class="form-control bgp-asn" type="number" required /></td>
+            <td><input class="form-control your-peer-ip" type="text" placeholder="198.162.1.1/24" pattern="^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/([0-9]|[1-2][0-9]|3[0-2]))$" required /></td>
```

Comment:
Since we have had to do this before, and we almost certainly will need to do this again.  Is there a way to build a module for the IPv4 and IPv6 regexp.  Seems like something we could write and then share... it should also be pretty basic